### ISSUE TITLE
No sound out

### ISSUE THREAD

**Issue by chris-roerig** [TomWhitwell/TuringMachine#77]
Everything appears to be working except for sound out. I'm getting noise from the noise output and clock etc all seem to be working. What part of the circuit manages the sound out? I'm not sure if its a bad IC or what. Thank you!

**Comment by Thonk-Steve** [comment]
Hi Chris, the Turing generates CV, not sound, what are you patching the
output to?

**Comment by chris-roerig** [comment]
Tom, thanks for the reply. After removing my palm from my face, I hooked up the "out" to the "vco 1v/oct" on my Mother 32. I've also run a gate to the TM. I have the VCA in full "on" mode but the TM is not altering pitch at all. What could be wrong here? I've replaced every IC and looked over every join on this thing. That doesn't mean its perfect but I'm running out of places to look. Thank you for your help!
